spec/lib/redmine/i18n_spec.rb

```diff
@@ -91,49 +91,57 @@ module OpenProject
 
       # it is OK if more languages exist
       it 'has multiple languages' do
-        expect(all_languages).to include :en, :de, :fr, :es
+        expect(all_languages).to include 'en', 'de', 'fr', 'es'
         expect(all_languages.size).to be >= 25
       end
     end
 
     describe 'valid_languages' do
       it 'allows only languages that are available' do
-        allow(Setting).to receive(:available_languages).and_return([:en])
+        with_settings(available_languages: ['en'])
 
-        expect(valid_languages).to eql [:en]
+        expect(valid_languages).to eq ['en']
       end
 
       it 'allows only languages that exist' do
-        allow(Setting).to receive(:available_languages).and_return([:'123'])
+        with_settings(available_languages: ['en', 'de', 'klingon'])
```

Comment:
Fun fact: Klingon is listed in [ISO 639-2 and has an official language identifier: `tlh`](https://en.wikipedia.org/wiki/Codes_for_constructed_languages)

Reply:
:laughing: Haha. I should have used a dummier string then. Thanks for sharing.